src/model/paste/__tests__/DraftPasteProcessor-test.js

```diff
@@ -203,6 +207,16 @@ describe('DraftPasteProcessor', function() {
     ]);
   });
 
+  it('must treat divs that do not contain Ps as Ps when we have Ps elsewhere', function() {
```

Comment:
In this test file `p` tags are of type `paragraph`. Before this change if the input HTML contained any tags with a type other than `unstyled` then all `div` tags (the default `unstyled` tag) would be collapsed.